gatsby-config.js

```diff
@@ -129,15 +129,6 @@ module.exports = {
     },
     "gatsby-plugin-sharp",
     "gatsby-transformer-sharp",
-    {
```

Comment:
@sangithravichandran dont commit unwanted files

Reply:
@PratheepV removed that file from the pull request.